core/templates/dev/head/pages/landing-page/topic-landing-page-stewards/topic-landing-page-stewards.mainpage.html

```diff
@@ -578,6 +578,7 @@ <h2>
 
 {% block footer_js %}
   {{ super() }}
-  <script src="/templates/dev/head/pages/landing/stewards/Stewards.js"></script>
+  <script src="/templates/dev/head/pages/landing-page/topic-landing-page-stewards/topic-landing-page-stewards.module.js"></script>
+  <script src="/templates/dev/head/pages/landing-page/topic-landing-page-stewards/topic-landing-page-stewards.controller.js"></script>
```

Comment:
This won't be necessary since you are already extending base.html so the scripts automatically are loaded.

Reply:
Also, the name of the file is to be topic-landing-page-stewards.directive.html as per the convention.

Reply:
ok

Reply:
done